### PR TITLE
feat: add default screen for the example

### PR DESCRIPTION
**Pull request by grabbou**
Pretty much similar to our home screen

### CODE REVIEW

File: example/demo/src/home/ui.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { navigate } from '@rise-tools/kit-react-navigation/server'
+import { Button, YStack } from '@rise-tools/kitchen-sink/server'
+
+export const models = {
+  '': Home,
```
Comment (ericvicenti) [inline example/demo/src/home/ui.tsx:5]: this feels strange. instead I think we should `export function Home` and assign the empty string to Home inside of `src/index`

**Review by ericvicenti — APPROVED** [review]
Nice! Great to have this

Please change the import thing before merging.

**Comment by ericvicenti** [comment]
Ok as we discussed, please add the <Rise> to show the list of examples

Then once you have a custom connection, have a greyed out hardcoded item that links to the new full screen home for the example server

**Comment by grabbou** [comment]
I am going to +1 this comment today to have evidence tomorrow I knew what we meant 😆

**Pull request merged**